examples/example-vite-svelte-recs/src/dojo/typescript/models.gen.ts

```
@@ -71,40 +71,40 @@
     direction: Direction;
-}
+};
 export const MovedDefinition = {
     player: RecsType.BigInt,
     direction: DirectionDefinition,
 };
 
 // Type definition for `dojo_starter::models::Vec2` struct
-export interface Vec2 {
+export type Vec2 = {
     x: Number;
```

Biome (1.9.4)

[error] Don't use 'Number' as a type.

Use lowercase primitives for consistency.
Safe fix: Use 'number' instead


(lint/complexity/noBannedTypes)